Yoast/wordpress-seo
# Hide Local notice when site representation is completed via the FTC

## Context


* This prepares the ground for Local's https://github.com/Yoast/wordpress-seo-local/pull/2553

## Summary


This PR can be summarized in the following changelog entry:

* Allows Local to hide its notice about organization info missing, when those are filled via the First Time Configuration.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

Together with https://github.com/Yoast/wordpress-seo-local/pull/2553:
* In a fresh site (or Reset options if not on a fresh site) install Free and Local
* Go to the General page and confirm that you get a `Missing organization name and logo` notice:
![image](https://github.com/user-attachments/assets/44f08880-2596-43e7-ac05-2f5145957cc8)
* Go to the FTC and in the `Site Representation` step, fill the Organization Name and the Organization Logo
* Click `Save and continue` and then go back to the Alert Center
* Confirm that the `Missing organization name and logo` notice is now gone
* Go back to the FTC and the `Site Representation` step and remove the logo
* Go back to the Alert center and confirm that the `Missing organization name and logo` notice is back
* Try the above but after finishing the FTC, so by clicking the `Site Representation`'s edit button.
* Also try the above but with Local's released version 
  * The `Missing organization name and logo` notice will not disappear without a page refresh, but at least confirm we are not getting console errors.

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC


* [] QA should use the same steps as above.


QA can test this PR by following these steps:

* Like the above instructions but ask for a Local zip, since we probably wont have an RC for that new change

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

**FTC completion notice**:
* In a fresh site, go to the General page and confirm you get a `SEO configuration` notice:
![image](https://github.com/user-attachments/assets/cacbf591-34e6-49ca-89b7-41e77ba60d9f)
* Go to the FTC tab and complete it
* Without refreshing, go to the Alert center and confirm that the `SEO configuration` notice is now gone

**Link parameters in webinar notice**:
* Confirm that the link of the webinar notice (`Sign up today!` - `https://yoa.st/webinar-intro-settings`) gets the same parameters as in the production version.

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes https://github.com/Yoast/wordpress-seo/issues/21769
